opengrok-indexer/src/main/java/org/opengrok/indexer/index/IndexDatabase.java

```diff
@@ -504,7 +508,11 @@ public void update() throws IOException {
 
                     args.cur_count = 0;
                     elapsed = new Statistics();
-                    LOGGER.log(Level.INFO, "Starting indexing of directory {0}", dir);
+                    if (dir.isEmpty()) {
+                        LOGGER.log(Level.INFO, "Starting indexing of directory 'source root directory'");
+                    } else {
+                        LOGGER.log(Level.INFO, "Starting indexing of directory {0}", dir);
```

Comment:
ditto

Reply:
@vladak 
Starting indexing of directory "source root directory"
Like this?

Reply:
The first statement is fine. I am after the second. It needs 2 single quotes actually, see the Java logging documentation. Or just try to run it...